display-scene-from-mobile-scene-package/README.metadata.json

```
@@ -0,0 +1,36 @@
+{
+    "category": "TODO",
+    "description": "Display a scene from a mobile scene package (.mspk)",
+    "formal_name": "DisplaySceneFromMobileScenePackage",
+    "ignore": false,
+    "images": [
+        "display-scene-from-mobile-scene-package.png"
+    ],
+    "keywords": [
+        "3d",
+        "mobile scene package",
+        "mspk",
+        "offline",
+        "scene",
+        "ArcGISScene",
+        "MobileScenePackage",
+        "SceneView"
+    ],
+    "language": "kotlin",
+    "provision_from": "TODO",
+    "provision_to": "TODO",
+    "redirect_from": "TODO",
```

You can leave an empty string here. 
```suggestion
    "redirect_from": "",
```

---

done, replaced the other two also with empty string.